stdlib/src/builtin/file.mojo

```diff
@@ -67,15 +68,16 @@ struct _OwnedStringRef(Boolable):
         return self.length != 0
 
 
+@value
 struct FileHandle:
     """File handle to an opened file."""
 
-    var handle: OpaquePointer
+    var handle: Arc[OpaquePointer]
```

Comment:
I don't like the idea of making this choice for users rather than them making the choice themselves. If a user does not need to pay the cost of an `Arc`, then they probably won't want to. I do think making it `Movable` has value though.